datascience-notebook/Dockerfile.onbuild

```diff
@@ -0,0 +1,7 @@
+FROM  jupyter/datascience-notebook
+
+ONBUILD COPY . /home/jovyan/work
+
+ONBUILD RUN  [ -e requirements.txt ] && pip install --no-cache-dir -r requirements.txt
```

Comment:
Probably should put in parentheses and add an `|| true` afterward to ensure the exit code is always `0`.